api/repositories.py

```diff
@@ -29,13 +29,14 @@ class UserRepository(Repository):
     def __init__(self, mongo_collection, repository_collection):
         Repository.__init__(self, mongo_collection, repository_collection)
 
-    def update_user_account(self,user_id,name,password,house_name,house_location):
+    def update_user_account(self,user_id,name,password_hash,house_name,house_location):
         user = self.get_user_by_id(user_id)
         if user is None:
-            return None
+            return False
```

Comment:
Why does this method return False here and a user object upon success?
I suggest you decide between a boolean return value (success/failure) or the up-to-date object upon succes and None upon failure.